packages/markdown-render-engine/package.json

```diff
@@ -39,12 +39,21 @@
     "@types/lodash.has": "^4.5.6",
     "@types/lodash.isdate": "^4.0.6",
     "@types/lodash.mapvalues": "^4.6.6",
-    "@types/react-copy-to-clipboard": "^4.2.6"
+    "@types/react-copy-to-clipboard": "^4.2.6",
+    "rollup": "^1.17.0",
```

Comment:
Was ist das? We already have rollup installed in root dir, so this is necessary? And below plugins?

Reply:
Rollup needs to be installed in every package in order to be able to build it, unfortunately it wouldn't see rollup in root.

`rollup-plugin-json` is there because there's some dependency package that is json
`rollup-plugin-node-builtins` and `rollup-plugin-node-globals` provide us with node's `util` and `event` packages, that are used by some dependency, and those are not present in browser environment, so they need to be provided that way